vm-setup/roles/v1aX_integration_test/vars/main.yml

```diff
@@ -13,6 +13,7 @@ CLUSTER_NAME: "{{ lookup('env', 'CLUSTER_NAME') | default('test1', true) }}"
 NUM_OF_MASTER_REPLICAS: "{{ lookup('env', 'NUM_OF_MASTER_REPLICAS') | default(1, true) }}"
 NUM_OF_WORKER_REPLICAS: "{{ lookup('env', 'NUM_OF_WORKER_REPLICAS') | default(1, true) }}"
 NUM_NODES: "{{ lookup('env', 'NUM_NODES') | default(2, true) }}"
+NODE_DRAIN_TIMEOUT: "{{ lookup('env', 'NODE_DRAIN_TIMEOUT') | default('0s', true) }}"
```

Comment:
Please define NODE_DRAIN_TIMEOUT in lib/common.sh taking the current value, and defaulting to 0 too.

Reply:
Done